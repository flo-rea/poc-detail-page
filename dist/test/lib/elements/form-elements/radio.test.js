"use strict";
const React = require('react');
const renderer = require('react/lib/ReactTestRenderer');
const radio_1 = require('../../../../src/lib/view/elements/form-elements/radio');
jest.mock('react-dom');
describe('Radio', () => {
    const component = renderer.create(React.createElement(radio_1.Radio, {id: "item2", name: "item2", checked: true}));
    it('has correct default options', () => {
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=radio.test.js.map