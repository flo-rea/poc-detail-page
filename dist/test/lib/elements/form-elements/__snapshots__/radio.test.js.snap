exports[`Radio has correct default options 1`] = `
<input
  checked={true}
  id="item2"
  name="item2"
  type="radio" />
`;
